lib/SPIRV/SPIRVInternal.h

```diff
@@ -1029,6 +1033,85 @@ bool postProcessBuiltinsReturningStruct(Module *M, bool IsCpp = false);
 
 bool postProcessBuiltinsWithArrayArguments(Module *M, bool IsCpp = false);
 
+/// \param Name LLVM function name
+/// \param DemangledName demanged name of the translator's internal built-in
+/// function.
+/// \returns true if Name is the name of the translator's internal built-in
+/// function, false for other functions
+/// Used for 'mini'-floats conversion functions
+bool isInternalSPIRVBuiltin(StringRef Name, StringRef &DemangledName);
```

Comment:
Also, based on usage of the function, please, consider renaming parameters. Just easier to read the code this way.
```suggestion
bool isInternalSPIRVBuiltin(StringRef MangledName, StringRef &BuiltinName);
```